ntfs_fr.md

```diff
@@ -1,5 +1,5 @@
-NTFS version 0.9.1
-==================
+NTFS version 0.10
```

Comment:
Then archive previous version?

As the format is being cleaned up currently, should we state on a clear semver use?
Or maybe target a `1.0.0` in short term then use semver?

Then choose when/how we want to archive versions and document all that?

Reply:
We decided to use github as an archive system, and only keep track of changes in the changelog. Maybe we can talk about this an other time ? (IMHO it's not related to this PR)